backend/chainlit/config.py

```diff
@@ -420,10 +424,7 @@ def load_settings():
                 "Your config file is outdated. Please delete it and restart the app to regenerate it."
             )
 
-        lc_cache_path = os.path.join(config_dir, ".langchain.db")
-
         project_settings = ProjectSettings(
```

Comment:
I believe you're missing something to change the `project_config["lc_cache_path"]` value, like:

```suggestion
        project_config["lc_cache_path"] = project_config.get("lc_cache_path", lc_cache_path)
        project_settings = ProjectSettings(
```

Reply:
the way i understand it and tested it locally, possible custom path will be picked up by `toml_dict` module [here](https://github.com/Chainlit/chainlit/blob/c153257fde517711d8d387de0f9add1ca925d810/backend/chainlit/config.py#L417C9-L417C54): 
``` python
project_config = toml_dict.get("project", {})
```
or will default to default value as proposed by my PR [here](https://github.com/Chainlit/chainlit/pull/982/files/c153257fde517711d8d387de0f9add1ca925d810#diff-e76edd7e411b9e044dbc1f307015efc5d8e83f7b254af919786191f86dfc7c74R287)
```python
# default to "{config_dir}/.langchain.db"
lc_cache_path: Optional[str] = os.path.join(config_dir, ".langchain.db")
 ```

Reply:
I think @necjamahim is right on this one. Could you please have another look @tpatel?